### PR TITLE
Video page cleanup

### PR DESCRIPTION
**Pull request by dbuenzli**
I fell on that dreadful page. It still will be, but I tried to improve it a bit.

### CODE REVIEW

File: site/community/media.md [hunk 1]
```diff
@@ -4,22 +4,24 @@
...
 
-The [OCamlorg YouTube account](http://www.youtube.com/user/ocamlorg)
-gathers several videos about OCaml, including videos of [all OUD
-presentations](http://www.youtube.com/playlist?list=PLOWQ4b7rjycjjAdv6uLy9SmzkvNzSY3HY).
+Many videos about OCaml can be found on
+[vimeo](https://vimeo.com/search?q=ocaml) or
+[youtube](https://www.youtube.com/results?search_query=ocaml).
+
+For OCaml beginners these
+[videos][https://vimeo.com/user20888710/videos] by John Whitington
```
Comment (pw374) [inline site/community/media.md]: I suspect it's not what you meant to write: isn't it () instead of [] ?

File: site/community/media.md [hunk 2]
```diff
@@ -4,22 +4,24 @@
...
+explain basic ideas about OCaml. They are good companion material to
+his [books][http://ocaml-book.com/] (the videos do not require or depend
```
Comment (pw374) [inline site/community/media.md]: I suspect it's not what you meant to write: isn't it () instead of [] ?

**Comment by dbuenzli** [comment]
@pw374 Indeed. Corrected.

**Comment by pw374** [comment]
thanks

**Pull request merged**